Makefile

```diff
@@ -86,6 +91,9 @@ clean:
 	rm -rf test/depth/basic/*.out
 	rm -rf test/depth/subset-paths/*.out
 
+	rm -rf test/handmade/*.crush
```

Comment:
Just pointing out where `rm` commands have been added to `clean`. Could we do this for the other files that we want `make clean` to remove?

Reply:
Ah sorry that my original question in https://github.com/cucapra/pollen/pull/50#issuecomment-1507803443 was not very clear. I _want_ `clean` to not delete those gfa files, since they are hand-rolled and don't live anywhere but this repository. So stashing them in handmade/ is my way of shielding them from `make clean`